### ISSUE TITLE
Pulumi: Issues with guacamole login

### ISSUE THREAD

**Issue by craddm** [alan-turing-institute/data-safe-haven#1607]
After a period of inactivity, Guacamole no longer offers a login page. Instead, you simply receive an error message:

![guac-error](https://github.com/alan-turing-institute/data-safe-haven/assets/5796417/7d95d783-c506-4de0-a326-f56a67141f93)

On inspecting the container logs, a standout error was this:

```
[http-nio-8080-exec-9] WARN o.a.i.d.pooled.PooledDataSource - Execution of ping query 'SELECT 1' failed: An I/O error occurred while sending to the backend.
```

This error led me to an issue on the Guacamole site, [https://issues.apache.org/jira/browse/GUACAMOLE-919], which suggests the problem is due to the connection between guacamole and the PostGRES database backing it. 

Manually changing the suggested environment variable on the running container seems to fix the issue - will give it a little longer to see if it recurs.

**Comment by jemrobinson** [comment]
Can you make a PR to add this environment variable setting to `data_safe_haven > infrastructure > stacks > sre > remote_desktop.py` in the `environment_variables` section for the `guacamole/guacamole` `containerinstance`?

**Comment by craddm** [comment]
Fixed in #1608
